test/components/previews/selection_preview/default.html.erb

```diff
@@ -0,0 +1,11 @@
+<div data-controller="selection">
```

Comment:
For this and the preview file below can I get you to add a few more items with checkboxes

Reply:
Please see c182d2c3bbcf38c96a7dcd8f3e6ff1880d3c5124.